app/src/main/java/com/example/paritosh/stormy/HourlyForecastActivity.java

```diff
@@ -0,0 +1,28 @@
+package com.example.paritosh.stormy;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+
+import com.example.paritosh.stormy.adapter.HourlyForecastAdapter;
+import com.example.paritosh.stormy.model.HourlyForecastModel;
+
+import java.util.ArrayList;
+
+public class HourlyForecastActivity extends AppCompatActivity {
+
+    RecyclerView recyclerView;
+    ArrayList<HourlyForecastModel> hourlyData;
```

Comment:
Why is this a global reference for this class. It should be a local reference as you are only using it once. Please remove this.